Add mode documentation and example in README (#33)
komposable/dekorator: README.md

decorated_user.posts.first # => PostDecorator
```

### Custom decorator

> By default dekorator search the decorator class by adding `Decorator` at the end.  

Reviewer: ```suggestion
By default, Dekorator searches for the decorator class by adding `Decorator` at the end.  
```

> For `User`, dekorator looks for `UserDecorator` class. And so for `User::Profile`

Reviewer: ```suggestion
For `User`, Dekorator looks for the `UserDecorator` class, and for `User::Profile`
```

end
```

## Testing

> `rails generate decorator user` also generate a testing file based on your

Reviewer: ```suggestion
`rails generate decorator user` also generates a testing file based on your
```

> `rails generate dekorator:decorator user` also generate a testing file based on your

Reviewer: ```suggestion
`rails generate dekorator:decorator user` also generates a testing file based on your
```
